### PR TITLE
Occupation Description

### PR DESCRIPTION
**Pull request by EmmieBunnie**
## Occupation Description Feature
Migrated a new description column to the preferences table. Created a conditional display of description text on the trade page of npcs. Seeded in a few more preferences and their descriptions.

### Tasks
- [X] Cucumber definitions ane steps (1hr)
- [X] Migrate description column (0.1hr)
- [X] Display tips of occupations on trade menu (1hr)
- [X] Come up with professions and their hhints (0.1hr)

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -117,7 +117,12 @@
   ShoppingList.find_or_create_by!(shoppinglist)
 end
 
-preferences = [{ occupation: 'merchant', item:'map', multiplier: 2}]
+preferences = [{ occupation: 'merchant', item:'map', multiplier: 2, 
+                 description: 'Grinds wheat into flour and bakes handbaked bread.'},
```
Comment (KiranWells) [inline db/seeds.rb]: This seems like it might be incorrect

File: app/views/trade/trade.html.erb [hunk 2]
```diff
@@ -3,6 +3,15 @@
 
 <h2><%= @character.dialogue_content %></h2>
 <div>
+  <p>
+    <strong>Occupation Description:</strong>
+    <% if not @pref.nil? %>
+        <%= @pref.description %>
+    <% else %>
+        This occupation does not have a preference.
```
Comment (KiranWells) [inline app/views/trade/trade.html.erb]: We probably just want to leave this blank if there is no description (no **Occupation Description:** label either)

**Review by KiranWells — CHANGES_REQUESTED** [review]
There are a couple of changes I mentioned, and the tests need to be updated for auth. Aside from that, looks good

**Comment by EmmieBunnie** [comment]
resolved merge conflict and fixed issues mentioned

**Review by KiranWells — APPROVED** [review]
looks good

**Pull request merged**